package.json

```diff
@@ -34,7 +34,8 @@
     "tailwindcss-animate": "^1.0.7",
     "typescript": "5.1.3",
     "vaul": "^0.9.0",
-    "zod": "^3.23.5"
+    "zod": "^3.23.5",
```

Comment:
Why are we installing zod package?

Reply:
@choir27 I believe Zod was added prior to this PR. Zod is a TypeScript-first schema declaration and validation library. The only changes I made where in line 38 where I added in Zustand. I hope this clarifies everything :)